client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -91,8 +95,7 @@ class EmailProvidersComparison extends React.Component {
 		gSuiteProduct: PropTypes.object,
 		isGSuiteSupported: PropTypes.bool.isRequired,
 		productsList: PropTypes.object.isRequired,
-		selectedSiteId: PropTypes.number,
-		selectedSiteSlug: PropTypes.string,
+		selectedSite: PropTypes.object,
```

Comment:
Since we need a `selectedSite` object for the header now, I've removed the `selectedSiteId` and `selectedSiteSlug` props since this data can be inferred from `selectedSite`.